### PR TITLE
app: view: show: truncate summary length

### PR DESCRIPTION
**Pull request by SantiagoTorres**
The form under bugs.archlinux.org has a maximum field length of 200. If
the length of the summary field computed is larger than this, the
propulated form will fail to submit, which requires the user (submitter)
to manually edit the summary field to have a shorter value.

Add a check to the get_bug_data function to use an alternative summary
if this length is exceeded.

(I personally had this issue when submitting the [zziplib](https://bugs.archlinux.org/task/53133) CVE's)

### CODE REVIEW

File: app/view/show.py [hunk 1]
```diff
@@ -19,6 +19,9 @@
 
 
 def get_bug_data(cves, pkgs, group):
+
+    MAXLENGTH_SUMMARY = 200
```
Comment (jelly) [inline app/view/show.py]: Would be nice add a small comment why the summary needs to cut to exactly 200.

Comment (SantiagoTorres) [inline app/view/show.py]: Ok, should it be as simple as `# Set the max length of the summary as the one defined in bugs.archlinux.org's form` or so?

Comment (jelly) [inline app/view/show.py]: `# the form's summary of bugs.archlinux.org has a maximum field length of 200` should be fine

**Comment by SantiagoTorres** [comment]
@jelly done, lmk if this is what we need :-)

**Comment by anthraxx** [comment]
can you put the magic `MAXLENGTH_SUMMARY` into the `config/00-default.conf` file f.e. in the [tracker] section as `summary_length_max`.
That would be more convenient to change, plus it would be neat if a value of `0` could be interpreted as infinite.

**Comment by SantiagoTorres** [comment]
@anthraxx Let me know if this is what you had in mind :)

**Comment by anthraxx** [comment]
Nearly, if you use `config_tracker.getint` you automatically get an int value and can avoid casting `TRACKER_SUMMARY_LENGTH_MAX` to int (as it already will be one) and use it directly :-)

**Comment by SantiagoTorres** [comment]
Oh, right! Just updated 👍

**Comment by anthraxx** [comment]
applied on cli

**Pull request closed without merging**